tests/test_upload_api.py

```
@@ -0,0 +1,219 @@
+"""
+Test cases for the Upload API endpoints.
+
+Tests the file upload functionality including:
+- PDF and video file uploads
+- File validation (size, type, etc.)
+- Status polling endpoints
+- Error handling
+"""
+
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)